sources/packages/web/src/views/aest/student/StudentAppealRequestsApproval.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Appeals"
+        sub-title="Appeal request"
+        :route-location="pendingAppealsRoute"
+      />
+    </template>
+    <student-appeal-requests-approval
```

Comment:
The component `student-appeal-requests-approval` should have provision to get the title and subtitle content from the consumer so that content changes in the future will be easily manageable.